src/main/java/com/entropyteam/entropay/employees/services/ContractService.java

```diff
@@ -44,7 +82,7 @@ protected BaseRepository<Contract, UUID> getRepository() {
     }
 
     @Override
-    protected ContractDto toDTO(Contract entity) {
+    public ContractDto toDTO(Contract entity) {
```

Comment:
could it be protected?

Reply:
I'm using this method in the test classes, that's why I changed it. Is there a better way to reuse this?